woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1448,3 +1448,76 @@ def test_pop_error(sample_df):
 
     with pytest.raises(LookupError, match="Column with name missing not found in DataFrame"):
         sample_df.ww.pop("missing")
+
+
+def test_accessor_drop(sample_df):
+    xfail_dask_and_koalas(sample_df)
+
+    original_columns = sample_df.columns.copy()
+    schema_df = sample_df.copy()
+    schema_df.ww.init()
+
+    single_input_df = schema_df.ww.drop('is_registered')
+    assert len(single_input_df.ww.columns) == (len(original_columns) - 1)
```

Comment:
What do you think about this type of assertion instead:
`assert len(single_input_df.ww.columns) == (len(schema_df.columns) - 1)`? 

I think this would also help us confirm that the `schema_df` is not being modified by the drop operation, which we wouldn't identify by comparing against a list that is a copy of the original columns in `sample_df`.

Reply:
Good point--I think it makes sense to make this change here and the other places below